api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -374,6 +382,12 @@ type Proxy struct {
 	NoProxy string `json:"noProxy,omitempty"`
 }
 
+// ManifestsConfigMapReference is a reference to a manifests ConfigMap
```

Comment:
Curious why this type is needed. It's the same as LocalObjectReference, except that the Name field is mandatory. Is that the only reason?

Reply:
okay.